scripts/deploy.sh

```diff
@@ -66,13 +66,28 @@ if [ "${DEPLOY_AIO}" != false ]; then
     #                  consume them.
     openstack-ansible -i 'localhost,' \
                       -e 'apt_target_group=localhost' \
+                      -e 'apt_artifact_mode="${RPCO_APT_ARTIFACTS_MODE}"' \
                       -e 'container_artifact_enabled=false' \
                       -e 'py_artifact_enabled=false' \
                       "${SCRIPT_PATH}/../playbooks/site-artifacts.yml"
 
     # Install OpenStack-Ansible
     openstack-ansible "${SCRIPT_PATH}/../playbooks/openstack-ansible-install.yml"
 
+    # Set the AIO config bootstrap options
+    apt_artifact_enabled=$(awk '/apt_artifact_enabled/ {print $3}' /etc/ansible/facts.d/rpc_openstack.fact)
+    if [[ "${apt_artifact_enabled}" == "True" ]]; then
+      if [[ "${RPCO_APT_ARTIFACTS_MODE}" == "strict" ]]; then
+        # Prevent the AIO bootstrap from re-implementing
+        # the updates, backports and UCA sources.
+        export BOOTSTRAP_OPTS='{ "bootstrap_host_apt_distribution_suffix_list": [], "uca_enable": "False" }'
+      else
+        # Prevent the AIO bootstrap from re-implementing
+        # the UCA sources.
+        export BOOTSTRAP_OPTS='uca_enable="False"'
```

Comment:
Does this export also need to be wrapped in braces?

Reply:
No, the braces above are only due to the expression of the list value. This is purely string value so it's not necessary.